src/sentry/testutils/fixtures.py

```diff
@@ -787,3 +788,14 @@ def create_container_with_samples(
             container.append(sample)
         container.save()
         return container
+
+
+def has_context(func):
```

Comment:
This does not appear to be used?

Reply:
Right! I added it so one can add a context around a test function without the test having a context. I then went with having it defined in the core test case as it's more convenient.